src/app/zone-ingresses/views/item/DetailView.vue

```diff
@@ -59,15 +59,24 @@
           </template>
         </KCard>
 
-        <div v-if="(data.zoneIngressInsight?.subscriptions ?? []).length > 0">
-          <h2>{{ t('zone-ingresses.detail.subscriptions') }}</h2>
+        <template
+          v-for="subscriptions in [props.data.zoneIngressInsight?.subscriptions ?? []]"
+          :key="subscriptions"
+        >
+          <div
+            v-if="subscriptions.length > 0"
+          >
+            <h2>{{ t('zone-ingresses.routes.item.subscriptions.title') }}</h2>
 
-          <KCard class="mt-4">
-            <template #body>
-              <SubscriptionList :subscriptions="data.zoneIngressInsight?.subscriptions ?? []" />
-            </template>
-          </KCard>
-        </div>
+            <KCard class="mt-4">
+              <template #body>
+                <SubscriptionList
+                  :subscriptions="subscriptions"
+                />
+              </template>
+            </KCard>
+          </div>
+        </template>
```

Comment:
Same simplification for ingress subscriptions.

Reply:
I’m a little confused. What’s the simplification here? Adding this "v-for to introduce template-scoped variable" hack is something we should really not use that liberally.

Reply:
Yeah if you look at the removed code here, we were doing the same multiple existence checks twice, now we just do it once.